start-demo.sh

```diff
@@ -25,14 +25,7 @@ DOCKER_SERVER_VERSION_MAJOR=$(echo "$DOCKER_SERVER_VERSION"| cut -d'.' -f 1)
 DOCKER_SERVER_VERSION_MINOR=$(echo "$DOCKER_SERVER_VERSION"| cut -d'.' -f 2)
 DOCKER_SERVER_VERSION_BUILD=$(echo "$DOCKER_SERVER_VERSION"| cut -d'.' -f 3)
 
-if [ "${DOCKER_SERVER_VERSION_MAJOR}" -ge 20 ] && \
-   [ "${DOCKER_SERVER_VERSION_MINOR}" -ge 10 ]  && \
-   [ "${DOCKER_SERVER_VERSION_BUILD}" -ge 13 ]; then
-    echo "Docker version >= 20.10.13, using Docker Compose V2"
-    if command -v gp version &> /dev/null; then
-        export PROXY_TLS="-DgitPodEnvironment"    
-    fi
-    docker compose up -d --build
-else
-    echo "Docker versions < 20.10.13 are not supported" 
```

Comment:
@kdaud but why remove the check on Docker versions?

Reply:
@enyachoke could you throw a comment here?

Reply:
> @kdaud but why remove the check on Docker versions?

Removing the check seemed to resolve the issue with gitpod however, we indeed need the logic. So @Ruhanga suggested to make the check robust which is working well on gitpod environment.